x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/policies/package_policies.tsx

```diff
@@ -97,16 +120,36 @@ export const PackagePoliciesPage = ({ name, version }: PackagePoliciesPanelProps
           defaultMessage: 'Agents',
         }),
         truncateText: true,
-        align: 'right',
+        align: 'left',
         width: '8ch',
         render({ packagePolicy, agentPolicy }: PackagePolicyAndAgentPolicy) {
+          const count = agentPolicy?.agents ?? 0;
+
           return (
-            <LinkedAgentCount
-              count={agentPolicy?.agents ?? 0}
-              agentPolicyId={packagePolicy.policy_id}
-              className="eui-textTruncate"
-              data-test-subj="rowAgentCount"
-            />
+            <>
+              <LinkedAgentCount
+                count={count}
+                agentPolicyId={packagePolicy.policy_id}
+                className="eui-textTruncate"
+                data-test-subj="rowAgentCount"
+              />
+              {count === 0 && (
```

Comment:
Easy change to make to remove this condition, but just want to call out where I saw this in the designs:

![Screen Shot 2021-06-08 at 11 08 28 AM](https://user-images.githubusercontent.com/6766512/121210743-eaa85400-c849-11eb-8a3f-a4b3c43effed.png)

Reply:
@hbharding can you clarify if this should always be shown or only when zero agents?

Reply:
@kpollich actually has this right. I had considered having a persistent action there when agents exist, but it felt problematic to have a text link + icon button side-by-side (it looked like a single link). The "plus" icon is more of a call to action for when no agents exist, in case users dismiss the toast which instructs users to add an agent. If agents already exist, users can find the "add agent" option in the "actions" button.

Reply:
Thanks Henry! Going to resolve this convo.